.env

```diff
@@ -1 +1 @@
-VUE_APP_PATH = basePath
+VUE_APP_PATH = businesses/create
```

Comment:
This is now the same as in `.env.production`.

Also, this might reduce confusion between Create and Edit UI.